packages/btcindexer/src/btcindexer.test.ts

```diff
@@ -102,10 +102,19 @@ beforeEach(async () => {
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	const env = (await mf.getBindings()) as any;
 	const storage = new CFStorage(env.DB, env.btc_blocks, env.nbtc_txs);
+	const nbtcAddressesMap = new Map<string, NbtcAddress>();
+	const testNbtcAddress: NbtcAddress = {
```

Comment:
Do we need a map here ? Or list is enough? We are using it as a list of values in line 105

Reply:
we are using map in the indexer for lookups, here in the test we have just one, but in the future it would be much eaiser when we have many different deposit addresses with their corresponding packages